Test/Integration/Export/BlueprintExportDataTest.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace StoreKeeper\StoreKeeper\Test\Integration\Export;
+
+use StoreKeeper\StoreKeeper\Helper\Base36Coder;
+use StoreKeeper\StoreKeeper\Test\Integration\AbstractTest;
+use Magento\TestFramework\Helper\Bootstrap;
+
+class BlueprintExportDataTest extends AbstractTest
+{
+    protected $blueprintExportManager;
+    protected  $csvFileContent;
+    protected  $json;
+    protected $base36Coder;
+
+    protected function setUp(): void
+    {
+        $this->blueprintExportManager = Bootstrap::getObjectManager()->create(\StoreKeeper\StoreKeeper\Model\Export\BlueprintExportManager::class);
+        $this->csvFileContent = Bootstrap::getObjectManager()->create(\StoreKeeper\StoreKeeper\Model\Export\CsvFileContent::class);
+        $this->json = Bootstrap::getObjectManager()->create(\Magento\Framework\Serialize\Serializer\Json::class);
+        $this->base36Coder = Bootstrap::getObjectManager()->create(Base36Coder::class);
+    }
+    /**
+     * @magentoConfigFixture current_store storekeeper_general/general/storekeeper_shop_language nl
+     * @magentoConfigFixture current_store general/locale/code nl_NL
+     * @magentoDataFixture StoreKeeper_StoreKeeper::Test/Integration/_files/configurable_products_with_two_attributes.php
+     */
+    public function testGetBlueprintExportData()
+    {
+        $this->assertEquals($this->getTestBlueprintExportData(), $this->getBlueprintCsvContent());
+    }
+
+    /**
+     * @return array
+     * @throws \Exception
+     */
+    public function getBlueprintCsvContent(): array
+    {
+        $csvData = $this->csvFileContent->getFileContents('blueprint');
+        $arrayData = str_getcsv($csvData, "\n");
+
+        return $arrayData;
+    }
+
+    /**
+     * @return array
+     * @throws \Magento\Framework\Exception\NoSuchEntityException
+     */
+    public function getTestBlueprintExportData(): array
+    {
+        return [
+            0 => 'path://name,path://alias,path://sku_pattern,path://title_pattern,path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_synchronized,path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_synchronized,path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_synchronized',
```

Comment:
```suggestion
            0 => [
                'path://name',
                'path://alias',
                'path://sku_pattern',
                'path://title_pattern',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_configurable',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_synchronized',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_configurable',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_synchronized',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_configurable',
                'path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_synchronized'
            ]
```

I meant something like this